packages/fxa-settings/src/pages/Pair/Index/index.tsx

```diff
@@ -0,0 +1,178 @@
+    entryPoint === ENTRYPOINTS.FIREFOX_TABS_SIDEBAR_ENTRYPOINT ||
+    entryPoint === ENTRYPOINTS.FIREFOX_FX_VIEW_ENTRYPOINT ||
+    entryPoint === ENTRYPOINTS.FIREFOX_TOOLBAR_ENTRYPOINT;
+
+  return (
+    <>
+      {showQRCode ? (
+        <CardHeader
+          headingTextFtlId="pair-cad-header"
+          headingText="Connect Firefox on another device"
+        />
+      ) : (
+        <CardHeader
+          headingTextFtlId="pair-sync-header"
+          headingText="Sync Firefox on your phone or tablet"
+        />
+      )}
+
+      <section>
+        {error && (
+          <Banner type={BannerType.error}>
+            <p>{error}</p>
+          </Banner>
+        )}
+        {showQRCode ? (
+          <>
+            <div className="mt-9 mb-5">
+              <FtlMsg id="pair-already-have-firefox-paragraph">
+                <p className="text-sm">
+                  Already have Firefox on a phone or tablet?
+                </p>
+              </FtlMsg>
+            </div>
+            <div className="flex">
+              <FtlMsg id="pair-sync-your-device-button">
+                <button
+                  className="cta-primary cta-xl"
+                  type="button"
+                  onClick={() => {
+                    onSubmit();
+                  }}
+                >
+                  Sync your device
+                </button>
+              </FtlMsg>
+            </div>
+            <FtlMsg id="pair-or-download-subheader">
+              <h2 className="text-lg font-bold mt-10">Or download</h2>
+            </FtlMsg>
+            <FtlMsg
+              id="pair-scan-to-download-message"
+              elems={{
+                linkExternal: (
+                  <LinkExternal
+                    href="https://www.mozilla.org/firefox/mobile/get-app/"
+                    className="link-blue"
+                  >
+                    download link.
+                  </LinkExternal>
+                ),
+              }}
+            >
+              <p className="my-5 px-16 text-sm">
+                Scan to download Firefox for mobile, or send yourself a{' '}
+                <LinkExternal
+                  href="https://www.mozilla.org/firefox/mobile/get-app/"
+                  className="link-blue"
+                >
+                  download link.
+                </LinkExternal>
+              </p>
+            </FtlMsg>
+            <div
+              className="mx-auto w-48 h-48"
+              role="img"
+              aria-label={localizedQRCodeLabel}
+            ></div>
+            <p className="mt-5 text-sm">
+              <FtlMsg id="pair-not-now-button">
+                <Link className="link-blue" to="/settings">
+                  Not now
+                </Link>
+              </FtlMsg>
+            </p>
+          </>
+        ) : (
+          <>
+            <HeartsVerifiedImage />
+            <FtlMsg id="pair-take-your-message">
+              <p className="mt-5 mb-5 px-16 text-sm">
```

Comment:
tiny nit: Since `mt-5` and `mb-5` use the same value, i think you can probably just use `my-5`.
Although now I'm wondering if this is a greppable question.

I managed to do this, but not sure how/if you can capture the first `[[:digit:]]` value easily and pass to the second <kbd>grep</kbd> to find places where they match.

```sh
git grep "mt-[[:digit:]]" | grep "mb-[[:digit:]]"
```